### PR TITLE
fix: build on Windows

### PR DESCRIPTION
**Pull request by smira**
Also update Kres and generated files.

Wireguard UAPI on Windows is different than UNIX one.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: pkg/wireguard/wireguard.go [hunk 1]
```diff
@@ -77,7 +69,7 @@ func (dev *Device) Run(ctx context.Context, logger *zap.Logger, peers PeerSource
 		Errorf:   logger.Sugar().Errorf,
 	}
 
-	uapi, err := ipc.UAPIListen(interfaceName, dev.fileUAPI)
+	uapi, err := UAPIOpen(interfaceName)
```
Comment (frezbo) [inline pkg/wireguard/wireguard.go:72]: so go automatically picks the right method definition based on os?

**Review by frezbo — APPROVED** [review]

File: pkg/wireguard/wireguard.go (hunk 1 above)
Comment (smira) [reply pkg/wireguard/wireguard.go:72]: yep, it build either one of the files wireguard_*.go based on the `GOOS`

Comment (frezbo) [reply pkg/wireguard/wireguard.go:72]: oh nvm, just saw the build tags

**Review by Unix4ever — APPROVED** [review]

**Pull request merged**